WordPress/src/main/res/values/strings.xml

```diff
@@ -599,7 +595,68 @@
     <string name="older_month">Older than a month</string>
     <string name="error_notification_open">Could not open notification</string>
     <string name="ignore">Ignore</string>
-    <string name="push_auth_expired">The request has expired. Sign in to WordPress.com to try again.</string>
+    <string name="push_auth_expired">The request has expired. Sign in to WordPress.com to try again.</string><string name="notifications_updates_from_wordpress">Updates from WordPress.com</string>
+
+    <!-- Notification Settings -->
+    <string name="notification_settings">Notification Settings</string>
+    <string name="notifications_sights_and_sounds">Sights and Sounds</string>
+    <string name="your_sites">Your Sites</string>
+    <string name="notifications_account_emails_summary">We’ll always send important emails regarding your account, security, privacy, and purchase transactions, but you can get some fun extras, too!</string>
```

Comment:
I know I'm harping about wording, but this "fun extras" part sounds like insincere marketing-speak. Any chance we can drop that?